policy/policy.go

```diff
@@ -257,3 +343,15 @@ type PoliciesList []StagedPolicies
 func (l PoliciesList) IsDefault() bool {
 	return len(l) == 1 && l[0].IsDefault()
 }
+
+// WithDefaultAggregation updates the PoliciesList with default aggregation types
+func (l PoliciesList) WithDefaultAggregation() PoliciesList {
+	for _, sp := range l {
+		for j := range sp.policies {
```

Comment:
nit: try using public APIs e.g., sp.Policies() when possible, this reduces the blast radius in case we need to change the encapsulated fields in the future.